PF/Assignment4/Permutation.java

```diff
@@ -0,0 +1,25 @@
+import java.util.List;
+
+public class Permutation {
+
+	public static List<String> permuteString(String beginningString,
```

Comment:
ACTON: The problem asked for a single argument function but you have used 3. Rewrite the program using only a argument.